2024/incoming/perl-version-bumper.pod

```diff
@@ -0,0 +1,619 @@
+bundle is implicitly loaded (it only contains features that are
+backward-compatible with traditional Perl).
+
+This means that the "sensibly-written code from 20 years ago" mentioned
+above is very likely to just run unmodified, and simply work.
+
+As explained above, unguarded syntactic features are available from
+line 0.
+
+=head3 Line 1 semantics
+
+We've seen that C<use VERSION> automatically loads the corresponding
+feature bundle (and associated builtins). This single line of code enables
+all official features and disables all deprecated features for the given
+version of the Perl language.
+
+In other words, putting a C<use VERSION> line at the top (line 1) of a
+Perl script or module I<declares> which version of the I<Perl language>
+the code that follows is written under. And because the effect is lexical,
+a script can load other modules that declare they were written against
+a different version of the language, and everything works transparently.
+
+I strongly believe that C<use VERSION> should be the I<first line>
+of any Perl code.
+
+=head3 Declaring a baseline of the Perl language
+
+For decades, the first recommendation made to Perl beginners and people
+asking for help on a Perl forum was to add C<use strict; use warnings;>
+at the beginning of their code. This made Perl more helpful, as it
+would point to likely errors in programming (like undeclared variables
+or undefined values).
+
+Since Perl v5.36, these two statements are implicitly included via
+C<use VERSION>. It also enables the official features of that version
+of the language, and disables the deprecated ones.
+
+Therefore, C<use VERSION> helps define a good I<baseline> for the Perl
+language, so that the compiler can fully understand the code that follows.
+It's of course possible to include experimental features, or re-enable
+deprecated features (the latter is really not recommended), to fine-tune
+the specific dialect of Perl in which the code is written.
+
+Always use a I<stable> version number with C<use VERSION>. The point is to
+I<declare> which version of the Perl I<language> your code is targetting.
+
+Future versions of C<perl> implicitly promise that they will understand
+the dialect of Perl declared by C<use VERSION>, and that they will run
+it to the best of their abilities.
+
+=head2 The Once and Future Perl
+
+Every year, a new version of C<perl> comes out. The C<perl> binary is
+updated, and so is the Perl language. The Perl 5 Porters support the
+two most recent stable releases of Perl, which gives you a good reason
+to upgrade your binary. You don't have to upgrade the version of the
+language your code uses, though.
+
+If you use Perl in 2024, you don't have to write code like the dot-com
+bubble is about to burst. (Be wary of the AI bubble, though...)
+You can easily signal your code was written in the last decade,
+and enjoy the recent evolutions of the language.
+
+=head3 The new "use strict and warnings"
```

Comment:
To me, most of this section feels like duplication of things you've said earlier.  I think you should either delete the whole thing or just figure out which *specific* statements are new and then put them in previous sections.

Reply:
I dropped it, and move the information about support to the top.